src/Microsoft.AspNetCore.SignalR.Core/DynamicHub.cs

```diff
@@ -9,7 +9,7 @@ public class DynamicHub : Hub
 
         public new DynamicHubClients Clients
         {
-            get { return _clients ?? new DynamicHubClients(base.Clients); }
+            get => _clients ?? new DynamicHubClients(base.Clients);
```

Comment:
Should this return a new `DynamicHubClients` every time?

Reply:
No I think it should look like this https://github.com/aspnet/SignalR/blob/5b6e81387aa2a301149d9c827273379ee617d2cd/src/Microsoft.AspNetCore.SignalR.Core/Hub%60T.cs#L10-L20

Reply:
Ok, thought so.